src/components/MyObservations/MyObservationsPressable.js

```diff
@@ -0,0 +1,44 @@
+// @flow
+
+import { useNavigation } from "@react-navigation/native";
+import type { Node } from "react";
+import React from "react";
+import { useTranslation } from "react-i18next";
+import { Pressable } from "react-native";
+
+type Props = {
+  observation: Object,
+  children: any
+}
+
+const MyObservationsPressable = ( { observation, children }: Props ): Node => {
+  const navigation = useNavigation( );
+  const { t } = useTranslation( );
+  const unsynced = !observation.wasSynced( );
+
+  const navigateToObservation = ( ) => {
+    const { uuid } = observation;
+    if ( unsynced ) {
+      navigation.navigate( "ObsEdit", { uuid } );
+    } else {
+      navigation.navigate( "ObsDetails", { uuid } );
+    }
+  };
+
+  return (
+    <Pressable
+      onPress={navigateToObservation}
+      accessibilityRole="link"
+      accessibilityHint={unsynced
+        ? t( "Navigate-to-observation-edit-screen" )
+        : t( "Navigate-to-observation-details" )}
+      accessibilityLabel={t( "Observation-Name", {
+        scientificName: observation.name
```

Comment:
Did you mean `species_guess`? Observation doesn't have a `name` attribute. I would also add a TODO here to use the name that the viewer prefers. IMO, we need to make a helper related to DisplayTaxonName that fetches the primary and secondary names of a taxon given the current user's preferences, but that's probably out of scope here.

Reply:
not sure, this was pre-existing code. I'll change to `species_guess` and add a note.